internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,58 @@
 package daysteps
 
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+
+	daySplitData := strings.Split(datastring, ",")
+	if len(daySplitData) != 2 {
+		return errors.New("daySplitData != 2")
+	}
+	daySteps, err := strconv.Atoi(daySplitData[0])
+	if err != nil {
+		return err
```

Comment:
Ошибки лучше оборачивать
```suggestion
		return fmt.Errorf("invalid steps: %w", err)
```